### PR TITLE
Update README w/ more links, clearer language

### PR DESCRIPTION
**Pull request by jamiew**
Plus a note about when the `sickrage/sickrage` repository takeover happened

I also made a number of edits to the wiki for clarity: [recent edits list](https://github.com/SickChill/SickChill/wiki/Home/_history)

* edit1 (sidebar) https://github.com/SickChill/SickChill/wiki/Home/983bccbc96887e20ca3c0cf4ad4c206efc6c804d
* edit2 (main page text) https://github.com/SickChill/SickChill/wiki/Home/435e2ef450577cc24d8e142e5a4d851ff42a8f77

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -31,30 +32,40 @@ Automatic Video Library Manager for TV Shows. It watches for new episodes of you
...
 A full list can be found here: [Link](https://github.com/SickChill/SickChill/wiki/SickChill-Search-Providers)
 
 #### Special Thanks to:
-![image](https://rarbg.com/favicon.ico)[RARBG](https://rarbg.to)
-![image](https://torrentproject2.se/favicon.ico)[TorrentProject](https://torrentproject.se/about)
-![image](https://thepiratebay.org/favicon.ico)[ThePirateBay](https://thepiratebay.se/)
-![image](https://nzb.cat/favicon.ico)[NZB.cat](https://nzb.cat/)
-![image](https://nzbgeek.info/favicon.ico)[NZBGeek](https://nzbgeek.info)
-![image](https://raw.githubusercontent.com/SickChill/SickChill/master/gui/slick/images/providers/dognzb.png)[DOGnzb](dognzb.cr)
+![image](https://rarbg.com/favicon.ico)[RARBG](https://rarbg.to) <br/>
```
Comment (miigotu) [inline readme.md]: dont usev `<br/>` - Just put 2 spaces at the end of the line

File: readme.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-#### We will never mine bit-coin on your machine, charge for a "service" or to use the software.
-#### We also will never store your information, especially site logins to private trackers.
+
+#### Please do not confuse us with SickRageTV aka Sickrage.ca, which took over the `sickrage/sickrage` repository in Oct 2018
+#### We will never mine bitcoin on your machine, charge for a "service" or to use the software.
+#### We will never store your information, especially site logins to private trackers.
 #### We believe in honesty and loyalty and privacy.
 
-Automatic Video Library Manager for TV Shows. It watches for new episodes of your favorite shows, and when they are posted it does its magic.
+Automatic Video Library Manager for TV Shows. Watches for new episodes of your favorite shows, and sends them to your downloader whenever they're available.
```
Comment (miigotu) [inline readme.md]: Leave this line how it was, for legal reasons.

File: readme.md [hunk 3]
```diff
@@ -31,30 +32,40 @@ Automatic Video Library Manager for TV Shows. It watches for new episodes of you
...
 
+#### Installation/Setup
+
+Visit the [Installation & Configuration guides page](https://github.com/SickChill/SickChill/wiki/Installation-&-Configuration-Guides) on our wiki
+
 #### Dependencies
- To run SickChill from source you will need Python 2.7.10
 
-##### [SickChill Issue Tracker](https://github.com/SickChill/SickChill/issues)
+To run SickChill from source you will need Python 2.7.10
```
Comment (miigotu) [inline readme.md]: The `from source` part can be taken out, we always need python 2.7.10+ whether its a zip install or a git install.

**Review by miigotu — CHANGES_REQUESTED** [review]

**Comment by jamiew** [comment]
Was this closed as n/a, or just closed because needs updates? I can revisit your notes later this week

File: readme.md (hunk 1 above)
Comment (jamiew) [reply readme.md]: All sane text editors strip that out (and I noticed it was already missing) so I end up needing to force it. This and image-link syntax are horrible parts of markdown IMHO

**Comment by miigotu** [comment]
Just because you didnt respond =P

**Comment by miigotu** [comment]
@jamiew so whats going on =P

**Comment by jamiew** [comment]
@miigotu implemented your suggestions except for EOL double-spaces since text editors have been stripping that, should be gtg now

I have more changes I'd like to make here (and in the wiki) but prefer to keep PRs small and moving along

Thanks for your patience, mediacenter contributions are a weekend activity for me :)

**Review by miigotu — APPROVED** [review]

**Comment by miigotu** [comment]
Make sure future PR are against the develop branch please =P

**Comment by jamiew** [comment]
Ah will do

Any thoughts about switching from master/develop to master/stable or master/release?

**Comment by miigotu** [comment]
Master is release/stable.

**Pull request merged**